src/run.rs

```diff
@@ -121,13 +123,13 @@ pub async fn run(
                         retryable.push(FixtureResult::Retryable { file })
                     }
                     Ok(FixtureResult::Skippable | FixtureResult::Success) => (),
-                    Ok(FixtureResult::Unauthorized) => {
-                        return Err(())
-                    },
+                    Ok(FixtureResult::Unauthorized) => return Err(()),
                     Err(Errored { file, reason }) => {
                         println!("An error occurred in loading fixture {}: {}", file, reason)
                     }
                 }
+
+                thread::sleep(std::time::Duration::from_millis(wait));
```

Comment:
Are we sure about this?
This function is blocking and the docs say it shouldn't be used in async functions.

Reply:
I found out I'd better use `tokio::time::sleep`. Done in https://github.com/trento-project/photofinish/pull/23/commits/82227366a6b26b598073abcc174e4a10bed736fa